script/clean

```diff
@@ -24,7 +24,7 @@ SRC_DIR=$ROOT_DIR/engine
 
 # delete build directory
 echo "deleting build directory ($BUILD_DIR)"
-rm -r $BUILD_DIR
+rm -rf $BUILD_DIR
```

Comment:
Should probably put a guard around that to make sure it's not empty or root

Reply:
There. How's that?

Reply:
Thanks